CMakeLists.txt

```diff
@@ -410,7 +410,7 @@ endif (WITH_THREADS AND Threads_FOUND)
 
 set (TEST_SRC_DIR \"${CMAKE_CURRENT_SOURCE_DIR}\")
 
-configure_file (src/config.h.cmake.in config.h)
```

Comment:
Please put the config under `glog/config.h` and reference it as such.

Reply:
There is already a src/glog/ directory. Creating another glog/ directory under root will make it more confusing as to why some headers are under src/glog/ and some are under glog/ . I think glog_config.h is more readable IMHO.

Reply:
The `glog/config.h` should be put into the build directory.